foursight_core/app_utils.py

```diff
@@ -41,6 +41,8 @@
 logging.basicConfig()
 logger = logging.getLogger(__name__)
 
+# TODO/dmichaels/2022-08-04: Get the "foursight-cgap" value some other way,
+# as the package might not be "foursight-cgap" (might be for example just "foursight").
 version = pkg_resources.get_distribution('foursight-cgap').version
```

Comment:
Why is the package to call package_resources on not just received as a class variable of AppUtilsCore? Something like:

```
    APP_PACKAGE = None  # to be filled out by the inheriting class
    @classmethod
    def app_version(cls):
        return pkg_resources.get_distribution(cls.APP_PACKAGE).version
```
so that in cgap-portal you do:
```
class AppUtils(AppUtilsCore):

    APP_PACKAGE = 'foursight-cgap'
```

You never want the dependencies to go backward. Instead, just creates a downstream way for the callers to give you the information you need in time for you to use it.

Reply:
Right, thanks!